lib/widgets/molecules/profile_experience_and_education.dart

```diff
@@ -0,0 +1,180 @@
+import 'package:flutter/material.dart';
+import 'package:mm_flutter_app/constants/app_constants.dart';
+
+class ExperienceAndEducation extends StatelessWidget {
+  const ExperienceAndEducation({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const Padding(
+      padding: EdgeInsets.all(Insets.paddingMedium),
+      child: Column(
+        children: [
+          Row(children: [_Experience()]),
+          Row(children: [_Education()]),
+        ],
+      ),
+    );
+  }
+}
+
+class _Experience extends StatelessWidget {
+  static final List<_ExperienceItem> _exampleExperience = [
+    _ExperienceItem(
+      position: "Marketing director",
+      companyName: "SVK Group",
+      start: DateTime.utc(2021, 12),
+      location: "Cleveland, Ohio, USA",
+    ),
+    _ExperienceItem(
+      position: "Senior Marketing Manager",
+      companyName: "SVK Group",
+      start: DateTime.utc(2019, 6),
+      end: DateTime.utc(2021, 11),
+      location: "Cleveland, Ohio, USA",
+    ),
+    _ExperienceItem(
+      position: "Fonder",
+      companyName: "MC Consulting",
+      start: DateTime.utc(2016, 3),
+      end: DateTime.utc(2019, 5),
+      location: "Cleveland, Ohio, USA",
+    ),
+  ];
+
+  const _Experience();
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    final items = _exampleExperience
+        .map((exp) => [
+              Text(exp.position, style: theme.textTheme.labelMedium),
+              Text(exp.companyName),
+              Text("(${exp.dateRange}) · ${exp.timeRange}"),
+              Text(exp.location),
+              const SizedBox(height: Insets.paddingMedium),
+            ])
+        .expand((i) => i)
+        .toList();
+
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text("My experience", style: theme.textTheme.labelLarge),
```

Comment:
1. Please create a l10n variable for this string
2. The labelLarge style is too similar to the one being used inside each section, so it is hard to tell where one section ends and the other begins. The Figma mock uses titleMedium.

The same two thing apply to the 'My education' section.

Reply:
Done. This makes sense. But I wonder - how did you notice that Figma mock uses `titleMedium`?

Reply:
In the Figma mock you can check the properties of all components. I made a post about  this on our the team's chat on July 18. Here is an example:

![image](https://github.com/micromentor-team/mm-flutter-app/assets/25093428/b9dd0158-159b-4c02-83ed-ece8e39d7d34)

Reply:
Thanks! I've adjusted the text style everywhere. Will remember in the future.